reVX/rpm/rpm_output.py

```diff
@@ -532,23 +558,24 @@ def _get_excl_data(excl, lat_slice, lon_slice):
 
         Returns
         -------
-        excl_data : np.ndarray
-            Exclusions data flattened and normalized from 0 to 1 (1 is incld).
+        incl_data : np.ndarray
+            Inclusions data mask flattened and normalized
+            from 0 to 1 (1 is incld).
         """
 
         if isinstance(excl, (ExclusionMask, ExclusionMaskFromDict)):
-            excl_data = excl[lat_slice, lon_slice]
+            incl_data = excl[lat_slice, lon_slice]
         else:
             e = 'Cannot recognize exclusion type: {}'.format(type(excl))
             logger.error(e)
             raise TypeError(e)
 
         # infer exclusions that are scaled percentages from 0 to 100
-        if excl_data.max() > 1:
-            excl_data = excl_data.astype(np.float32)
-            excl_data /= 100
+        if incl_data.max() > 1:
```

Comment:
Start the train! Exclusions make no sense in hindsight!

Reply:
you mean incl vs. excl? Agreed.... when a method is called "something_exclusions" and returns a variable called "inclusions" you know something has been named poorly...